source/Nevermore.IntegrationTests/Legacy/FixtureWithRelationalStore.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using Nevermore.Contracts;
+using Nevermore.Mapping;
+using NUnit.Framework;
+
+namespace Nevermore.IntegrationTests.Legacy
```

Comment:
Copied the old integration tests to a `Legacy` subdir here. We could remove them if/when we want to only cover the new style of documents.